unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldAudioHandler.cs

```diff
@@ -89,7 +88,12 @@ public void EnterEditMode(ParcelScene scene)
         UpdateEntityCount();
 
         if (eventBuilderMusic.source.gameObject.activeSelf)
-            startBuilderMusicCoroutine = StartCoroutine(StartBuilderMusic());
+        {
+            if (startBuilderMusicCoroutine != null)
+                CoroutineStarter.Stop(startBuilderMusicCoroutine);
+
+            startBuilderMusicCoroutine = CoroutineStarter.Start(StartBuilderMusic());
```

Comment:
this is a monobehaviour, why are we using `CoroutineStarter`?

Reply:
I think I introduced this change to avoid issues with the disabled `MonoBehaviour` not being able to start coroutines.